internal/rbd/rbd_util.go

```diff
@@ -1675,3 +1675,21 @@ func (rs *rbdSnapshot) isCompatibleThickProvision(dst *rbdVolume) error {
 
 	return nil
 }
+
+// newLevelSpec is copy of NewLevelSpec implementation from
+// https://github.com/ceph/go-ceph/blob/master/rbd/admin/admin.go.
+// TODO once https://github.com/ceph/go-ceph/issues/522 is fixed remove this.
+// nolint
+func (ri *rbdImage) newLevelSpec() string {
```

Comment:
what is the difference with `rbdImage.String()`, is this function really needed?

Reply:
sightly different when rados namespace is not set not major. Looks like I over-engineered. this is not needed removing